promtail/config.yml

```diff
@@ -0,0 +1,37 @@
+server:
+  http_listen_port: 9080
+  grpc_listen_port: 0
+
+positions:
+  filename: /tmp/positions.yaml
+
+clients:
+  - url: https://$USER:$PASSWORD@loki.gcp.obol.tech/loki/api/v1/push
```

Comment:
We have this env var, which is technically the same endpoint: 
https://github.com/ObolNetwork/lido-charon-distributed-validator-node/blob/254f491e36280318e64813659d47be03efc1bb66/.env.sample.mainnet#L78